### DISCUSSION TITLE
How should I change the "filament/forms" template?

### DISCUSSION THREAD

**Discussion by sh4msi** [filamentphp/filament#5006] [Help]
**Summary: Pages created with livewire and Filament Form do not follow custom styling.**

I use the Filament panel, I created another page based on the default login page (with the help of livewire and Filament Form)

But unfortunately, on the contrary, the default login page does not follow the styling template that I created and loads the default styles from the main Filament package, **what is the solution for this issue?**

 I noticed that when I don't load the login from the main configuration ('login' => null) and call the same default login through routing, then the default login page doesn't follow my custom template and styling.

**Comment by danharrin** [comment]
Without code or a reproduction repo it's difficult to help you.

**Reply by sh4msi** [reply]
https://github.com/sh4msi/fortest

![image](https://user-images.githubusercontent.com/6229984/202907473-f493b491-09bf-4a5d-93ef-4d9fbd4cf2f4.png)
![image](https://user-images.githubusercontent.com/6229984/202907497-73e7656a-0660-4872-96ce-06c8a7847511.png)

**Reply by danharrin** [reply]
And where are you defining custom styling?

**Reply by sh4msi** [reply]
In the sample code, I added only the files described in the "Building themes" section of the documents
As it is clear in the photos, the page created with livewire and filament form does not follow the color scheme of the template.

**Reply by sh4msi** [reply]
In the original code, I used 2 methods for custom styles, neither of which worked

_in ServiceProvider_
```
            Filament::registerStyles([
                asset('css/custom.css'),
            ]);
```

resources/css/app.css
`@import '../../resources/css/custom.css';`

**Reply by danharrin** [reply]
I think you're looking to register a theme instead of registering styles, right? Because you'd need to replace our Tailwind build in order to change the colors. Follow the Building Themes docs

**Comment by sh4msi** [comment]
Maybe it's better to ask my question like this
**In pages where "filament/forms" are used independently, how can I change the format of "filament/forms"?**

In the sample code that I prepared for you, I acted exactly according to the "Building Themes" documents
And as it is clear in the photos sent, the template settings have no effect on the register page("filament/forms")

**Reply by danharrin** [reply]
If the form is outside the admin panel, your stylesheet is linked from within your own Blade layout file. Changes in your Tailwind config are managed by you, not Filament. I would suggest building a small project with the TALLstack without Filament first, and then introducing Filament. Otherwise you will find these Tailwind concepts confusing.
